Maple2.Model/Game/Field/FieldAccelerationStructure.cs

```diff
@@ -59,36 +62,33 @@ internal enum FieldEntityMembers : byte {
 */
 public class FieldAccelerationStructure : IByteSerializable, IByteDeserializable {
     public const int AXIS_TRIM_ENTITY_COUNT = 10;
+    public const float BLOCK_SIZE = (float) Constant.BlockSize;
+    public const float HALF_BLOCK = 0.5f * BLOCK_SIZE;
```

Comment:
This one's just referencing the constant one, because that one's a different type than needed here. I also went with defined locally, because `Constant.` everywhere would bloat the line sizes & reduce readability with how often it's used per line.

Reply:
makes sense